grabbit/src/main/groovy/com/twcable/grabbit/spring/batch/repository/JcrJobExecutionDao.groovy

```diff
@@ -39,7 +39,7 @@ import static org.apache.sling.api.resource.ResourceUtil.getOrCreateResource
  */
 @CompileStatic
 @Slf4j
-class JcrJobExecutionDao extends AbstractJcrDao implements JobExecutionDao {
+class JcrJobExecutionDao extends AbstractJcrDao implements GrabbitJobExecutionDao {
```

Comment:
Could a GrabbitJcrJobExecutionDao produce GrabbitJcrJobExecution objects that can then produce their respective GrabbitJcrStepExecution (s)?